include/kunit

```diff
@@ -167,7 +167,8 @@ need_kconfig:
 - SND_SOC_INTEL_AVS_MACH_I2S_TEST: m
 
 - KALLSYMS: y
-
+- V4L_TEST_DRIVERS: y
```

Comment:
looks no need to have this, as it builds a test driver not something that runs test.